app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/logItemActivity/historyItem/historyItem.jsx

```diff
@@ -62,10 +62,12 @@ export class HistoryItem extends Component {
     projectId: PropTypes.string.isRequired,
     filterId: PropTypes.string.isRequired,
     patterns: PropTypes.array,
+    onHistoryRelevantItemLinkClick: PropTypes.func,
   };
 
   static defaultProps = {
     patterns: [],
+    onHistoryRelevantItemLinkClick: () => {},
```

Comment:
Can we rename it here to just `onClick`?
As I see it, it is not necessary to know anything about the specific action coming from the props.

Reply:
Updated